code/modules/antagonists/wizard/equipment/spellbook_entries/assistance.dm

```diff
@@ -97,3 +97,13 @@
 	limit = 3
 	category = "Assistance"
 	refundable = TRUE
+
+/datum/spellbook_entry/item/vendormancer
+	name = "Staff of Vendormancy"
+	desc = "A staff with the teachings of the first Runic Vendormancer.\
+		It can summon up to 3 Runic Vendors that decay over time, but can be \
+		throw around to squash oponents or be directly detonated. When out of \
+		charges a long channel will restore the charges."
+	item_path = /obj/item/runic_vendor_staff
+	category = "Assistance"
+	refundable = TRUE
```

Comment:
I don't think you can refund staffs, can you?

Reply:
I will test it later to see if any issue pops up, but you can refund other items so I assume it would not be an issue.
But now that I checked the items, they are ghost spawn options which make sense for refunds so I think this should be changed either way.